redhat-appstudio/service-provider-integration-operator
# Extra check if response is not empty before reading

### What does this PR do?
{SUBJ}

### Screenshot/screencast of this PR
n/a

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/SVPI-636

### How to test this PR?
1. Create SPIFileContentRequest
2. Create RemoteSecret with data that contains new lines in username and password.
3. Expected error in the operator logs
